### PR TITLE
Upgrade chart js

### PR DESCRIPTION
**Pull request by djdeath**

### CODE REVIEW

File: assets/web/chart.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
 /*!
- * Chart.js v2.9.4
- * https://www.chartjs.org
- * (c) 2020 Chart.js Contributors
- * Released under the MIT License
+ * Chart.js
+ * http://chartjs.org/
+ * Version: 2.4.0
```
Comment (pilou-) [inline assets/web/chart.js:4]: Isn't `2.4.0` older than `v2.9.4` ? Besides the last release is [v3.4.1](https://github.com/chartjs/Chart.js/releases). This [minified version](https://cdn.jsdelivr.net/npm/chart.js@3.4.1/dist/chart.min.js) mentions 3.4.1 too.

**Review by pilou- — COMMENTED** [review]
Cheers

**Comment by conchyliculture** [comment]
Do you want to keep this up for an actual update of chart.js?

**Comment by conchyliculture** [comment]
see #41

**Pull request closed without merging**